tools/build_script_generator/module.lb

```diff
@@ -11,6 +11,17 @@
 # -----------------------------------------------------------------------------
 
 import os
+import platform
+
+#filter to windowsify paths separators
+def windowsify(path):
+	if 'Windows' in platform.platform():
+	    path = path.replace('/','\\')
```

Comment:
Is this a stray line? Cos above you only did this: `.replace('\\','\\\\')) #windows path compatibility hack`

Reply:
That line is needed because the path will be something like "path/to\\release.elf", because a normpath call is missing for the "path/to" part of the path. If you think it will be more clear I'll change it in a os.path.normpath(path).